### PR TITLE
Update 03.json

### PR DESCRIPTION
**Pull request by rioten**
Added Def Hacks() CA 2017 onto hackathon listing.

### CODE REVIEW

File: api/1.0/2017/03.json [hunk 1]
```diff
@@ -472,5 +472,24 @@
       "googlePlusURL": "",
       "notes": ""
     }
```
Comment (paked) [inline api/1.0/2017/03.json]: You're missing a comma here!

**Comment by julia-guo** [comment]
Can someone add MontyHacks to the site? It's in "Issues." Thanks!

**Comment by rodrigoargumedo** [comment]
Hey @julia-guo, as we get a lot of issues with submissions, we are working hard to get every hackathon listed on the website. Hang tight while we work on the backlog of submissions. Thanks for your contribution and time!

**Comment by rodrigoargumedo** [comment]
Closing the pull request since the changes have not been made after a while.

**Pull request closed without merging**